sledger.go

```diff
@@ -295,3 +301,45 @@ func doForward(db *sql.DB, index int, yamlForward string, yamlBackward string) {
 		panic(err)
 	}
 }
+
+func isFlagPassed(name string) bool {
+	found := false
+	flag.Visit(func(f *flag.Flag) {
+		if f.Name == name {
+			found = true
+		}
+	})
+	return found
+}
+
+func getDbFromEnv() string {
+	host := OSGetenv("DATABASE_HOST")
+	if len(host) < 1 {
+		fmt.Println("Host not specified. Default to localhost")
+		host = "localhost"
+	}
+
+	port := OSGetenv("DATABASE_PORT")
+	if len(port) < 1 {
+		fmt.Println("Port not specified. Default to 5432")
+		port = "5432"
+	}
+
+	dbName := OSGetenv("DATABASE_NAME")
+
+	dbUser := OSGetenv("DATABASE_MIGRATION_USERNAME")
+	if len(dbUser) < 1 {
+		fmt.Println("User not specified.")
+	}
+
+	fmt.Println("  Database:")
+	fmt.Println("  - Host: ", host)
+	fmt.Println("  - Port: ", port)
+	fmt.Println("  - Name: ", dbName)
+	fmt.Println("  - User: ", dbUser)
+
+	dbUserPass := OSGetenv("DATABASE_MIGRATION_PASSWORD")
+
+	dbEnv := ("postgresql://" + dbUser + ":" + dbUserPass + "@" + host + ":" + port + "/" + dbName + "?sslmode=disable")
```

Comment:
Recommend doing some `fmt.Sprintf` magic here.

Reply:
I mistook it for a `fmt.Println` and thought that would output to logs (which we don't want because passwords). Fixed with latest commit